portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppServiceImpl.java

```diff
@@ -3332,13 +3342,20 @@ protected void copyFolderDependencies(
 
 				long[] assetCategoryIds = serviceContext.getAssetCategoryIds();
 				String[] assetTagNames = serviceContext.getAssetTagNames();
+				Long fileEntryTypeId = ParamUtil.getLong(
+					serviceContext, "fileEntryTypeId");
```

Comment:
Why isn't this a primitive `long`?

Reply:
that is what happens when you let intelliJ extract the variable :D

Thanks for pointing at it.